code/drasil-lang/lib/Language/Drasil/NounPhrase.hs

```diff
@@ -270,5 +270,5 @@ capWords w@(c:cs)
 
 -- | Words that should not be capitalized in a title (prepositions, articles, or conjunctions).
 doNotCaps :: [String]
-doNotCaps = ["a", "an", "the", "at", "by", "for", "in", "of",
+doNotCaps = ["a", "an", "the", "at", "for", "in", "of",
```

Comment:
Hmm... if this is the only solution to capitalizing "By", it's probably worth reverting this and making the capitalization of "By" a separate issue. We would (likely) still want "by" to be lowercase in a normal title, but uppercase in our specific use of "Referenced By" in the table, so we would need some more design to figure out how to do that.

Reply:
Agreed!

Reply:
Ok, I will revert the change and create a separate issue for that.